### PR TITLE
chore: chart tooltips always using new tooltip [WEB-828]

### PR DESCRIPTION
**Pull request by mapmeld**
## Description

While working on the new chart, it was necessary to keep that tooltip work in its own file.  We've now advanced to the point we can replace the old tooltip component.

## Test Plan

Without the new charts flag:
- Test tooltip is working on a single trial experiment's metrics chart
- Test tooltip is working on a multi-trial experiment learning curve chart (might be helpful to test with a smaller number of trials, 10 trials)

## Checklist

- [x] Changes have been manually QA'd
- [x] User-facing API changes need the "User-facing API Change" label.
- [x] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [x] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Comment by mapmeld** [comment]
I think we have similar issues with current tooltip + too many series

The main changes that I remember:
- some fixes with color of series and circle
- ordering series in the tooltip from highest to lowest value
- bolding the series in the tooltip which are closest (on y-axis) to the cursor's y position

**Comment by keita-determined** [comment]
> I think we have similar issues with current tooltip + too many series

ah right. prob its inevitable.

File: webui/react/src/components/UPlot/UPlotChart/tooltipsPlugin.ts [hunk 1]
```diff
@@ -35,25 +40,64 @@ export const tooltipsPlugin = (
...
-        ${header ? header + '<br />' : ''}
-        ${xSerie.label}: ${xValue}
+        ${header}
+        ${xSerie.label} ${xValue}
       </div>`;
 
-    uPlot.series.forEach((serie, i) => {
-      if (serie.scale === 'x' || !serie.show) return;
+    let minYDist = 1000;
+    const seriesValues = uPlot.series
```
Comment (keita-determined) [inline webui/react/src/components/UPlot/UPlotChart/tooltipsPlugin.ts:48]: nice to have type for `seriesValues`

**Review by keita-determined — COMMENTED** [review]
mostly good, but when there's 1 data point, tooltip is always shown. i think this should be hidden when cursor is out of chart

image

File: webui/react/src/components/UPlot/UPlotChart/tooltipsPlugin.ts (hunk 1 above)
Comment (mapmeld) [reply webui/react/src/components/UPlot/UPlotChart/tooltipsPlugin.ts:48]: will do ✅

**Comment by mapmeld** [comment]
@keita-determined - added `closeOnMouseExit` prop for LearningCurve and TrialChart (the two old charts using tooltip)

**Review by keita-determined — APPROVED** [review]

**Pull request merged**